resqpy/grid_surface.py

```diff
@@ -1508,7 +1508,7 @@ def find_faces_to_represent_surface_regular(grid,
 
 
 @njit
-def where_true(data: np.ndarray) -> np.ndarray:
+def where_true(data: np.ndarray):
```

Comment:
Is this an intentional change? Is it required? The just-in-time compilation works best with full type information, as I understand it. (Admittedly in this case it probably doesn't make any difference – I am curious about the change though.)

Reply:
This change seemed to be required to get the `mypy` type checker job to pass. I think the type information provided by numpy was updated in the more recent versions, which conflicted with the given return value here.

Given than `np.where` is [natively supported](https://numba.pydata.org/numba-doc/dev/reference/numpysupported.html#other-functions) by numba, I think it should be fine to include within a jitted function even without a return type hint.

Reply:
By all means we could re-add the type information in a future change, if it can be done in a way that passes type checking